snippets/kotlin_mpp_lib/shared/src/commonMain/kotlin/com/example/kotlinmpplib/Metadata.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.kotlinmpplib
+
+import breez_sdk.*
+class Metadata {
+    fun SetPaymentMetadata(sdk: BlockingBreezServices) {
+        // ANCHOR: set-payment-metadata
+        try {
+            sdk.SetPaymentMetadata("target-payment-hash", """{"myCustomValue":true}""")
```

Comment:
```suggestion
            sdk.setPaymentMetadata("target-payment-hash", """{"myCustomValue":true}""")
```

I believe the method is called with lowercase.